full_name.rb

```diff
@@ -0,0 +1,22 @@
+class FullName
+  def initialize()
+    @name_array = []
+    print "What's your first name? "
+    @name_array.push(gets.chomp)
```

Comment:
Just so you know, you can add items to an array using a "shovel"

`@name_array << gets.chomp` would work. 

Nothing wrong with how you did it, but I tend to see `<<` more "out in the wild" so thought I'd mention it.

Reply:
Thank you. Tried shovel elsewhere in this assignment after completing this file.